docs/products/kafka/howto/configure-log-cleaner.rst

```diff
@@ -0,0 +1,30 @@
+Configure log cleaner for compaction
+=====================================
+
+By default the log cleaner is enabled, but the log compaction is disabled. To enable log compaction, follow these steps:
+
+#. Log into the `Aiven web console <https://console.aiven.io/>`_ and select your Aiven for Apache Kafka service.
+#. Scroll down to *Advanced configuration* and click on **Add configuration option**.
+#. Find ``log.cleanup.policy`` in the list and select it.
+#. Set the value to ``compact``.
+
+This change will affect all topics in the cluster that do not have a configuration override in place.
```

Comment:
I believe this needs to be a warning, since it can affect existing topics and create disasters